app/views/assignments/_form.html.erb

```diff
@@ -378,6 +378,16 @@
         <%= label_tag 'penalty_decay_period_submission_rule',
                       PenaltyDecayPeriodSubmissionRule.human_attribute_name(:form_description) %>
         </p>
+        <div id='penalty_type_selector_decay' class='penalty-type-selector table-with-add' style='display:none'>
+          <%= t('submission_rules.penalty_type') %>:
+          <%= rule.select :penalty_type,
+                          [
+                            [t('submission_rules.penalty_types.percentage'), 'percentage'],
```

Comment:
Oh, please use the `ExtraMark` constants here as well (not the string literals). Keep in mind that this is embedded Ruby code, so you have access to all of the backend models here!